### PR TITLE
Issue 103 python data fetching

### PR DESCRIPTION
**Pull request by davidthomas5412**
This pull requests sets up the data fetching and generally cleans up the structure of the project. It takes a reasonable departure from [PR 3](https://github.com/davidthomas5412/Pangloss/pull/3) and [PR 1](https://github.com/davidthomas5412/Pangloss/pull/1) - which will now be obsolete.

Summary of changes:
1) data.py adds data fetching capability.
2) top level scripts have been moved into the bin directory.
3) setup.py and requirements.txt have been added to allow pip installations
4) docs have been updated to be consistent with above changes

Testing:
1) Confirmed pip install works on local machine
2) Confirmed downloads work from local machine

Next Steps:
Ready to merge after @drphilmarshall reviews.

### CODE REVIEW

**Comment by davidthomas5412** [comment]
This PR corresponds to (Issue 103)[https://github.com/drphilmarshall/Pangloss/issues/103]. After it is merged we will close the issue.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+astropy>=1.2.1
```
Comment (drphilmarshall) [inline requirements.txt:1]: Do we need all these version numbers? And is the syntax really ">=" ? If so I can see how they are harmless.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from setuptools import setup
+
+setup(
+    name="pangloss",
+    version='0.0.1',
+    author="Phil Marshal",
```
Comment (drphilmarshall) [inline setup.py]: There are two 'l's in Marshall :-) We should list Tom Collett, Spencer Everett and David Thomas as well.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    author_email="pjm@slac.stanford.edu",
+    url="https://github.com/drphilmarshall/pangloss",
+    packages=["pangloss"],
+    description="Line of sight weak lensing and inference.",
+    long_description=open("README.md").read(),
+    package_data={"": ["README.md", "LICENSE"]},
+    include_package_data=True,
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "License :: OSI Approved :: GNU General Public License (GPL)",
```
Comment (drphilmarshall) [inline setup.py]: If we are not already using the MIT license we should switch, straight away.

**Review by drphilmarshall — CHANGES_REQUESTED** [review]
Minor revision needed, mostly this looks good! :-)

File: requirements.txt (hunk 1 above)
Comment (davidthomas5412) [reply requirements.txt:1]: The versions are the versions I have locally. I used >= to ensure that users wouldn't hit some issue from having an old package - although there is no evidence this is a problem. Sounds like you would like to get rid of the >= ?

File: setup.py (hunk 3 above)
Comment (davidthomas5412) [reply setup.py]: Ok, current license is GPL, so I will change it to MIT on this branch.

**Comment by drphilmarshall** [comment]
Great, thanks for changing the license! You might do a quick grep for "GPL"
to make sure all files are consistent with the new regime. And the README
may need updating too.

**Comment by davidthomas5412** [comment]
@drphilmarshall GPL references have been removed. Should be good to go.

**Review by drphilmarshall — APPROVED** [review]
Great! Thanks David

**Pull request merged**